commons-httpclient/src/main/java/org/apache/commons/httpclient/HttpMethodDirector.java

```diff
@@ -150,9 +152,14 @@ public void executeMethod(final HttpMethod method) throws IOException, HttpExcep
         
                 // get a connection, if we need one
                 if (this.conn == null) {
+                    if (connectionManager.getParams().getSoTimeout()>0) {
```

Comment:
Formatting issue? keep spaces before and after >

Reply:
Thanks for comment.
I format the code you mentioned.